### PR TITLE
Feature/new messages indicator

### LINKED ISSUE #154: New messages
Is it possible to show new messages DateSeparator and have a flag on each new message that is new? 

Example: I have a new message on a channel which i didn't open. When I open it I want to see the 'New messages' DateSeparator and those new messages to have a red background. (Similar to Slack's new messages separator + a flag for each message that is considered new)

### PR DESCRIPTION
**Pull request by mahboubii**
fixes #154

### CODE REVIEW

File: src/components/MessageList/MessageListInner.js [hunk 1]
```diff
@@ -49,7 +50,21 @@ const insertDates = (messages) => {
       prevMessageDate = messages[i - 1].created_at.toDateString();
     }
 
-    if (i === 0 || messageDate !== prevMessageDate) {
+    if (!unread) {
+      unread = lastRead.getTime() < message.created_at.getTime();
```
Comment (vicnicius) [inline src/components/MessageList/MessageListInner.js:54]: is `lastRead` always defined?

Comment (vicnicius) [reply src/components/MessageList/MessageListInner.js:54]: It looks like it isn't: https://github.com/GetStream/stream-chat-js/blob/ca4f5a587d7d10c6a424b82eba0a359605f0023a/src/channel.ts#L1110

```suggestion
      unread = lastRead && lastRead.getTime() < message.created_at.getTime();
```

Comment (mahboubii) [reply src/components/MessageList/MessageListInner.js:54]: logically it should not be undefined at this stage, but won't hurt to have an extra check 👍

**Review by jaapbakker88 — APPROVED** [review]
LGTM

**Review by vicnicius — APPROVED** [review]

File: src/components/MessageList/MessageListInner.js (hunk 1 above)
Comment (vicnicius) [reply src/components/MessageList/MessageListInner.js]: And will make TS less angry in the future x)

**Pull request merged**